packages/ui/src/viewer-table/DataTable/TableRow.tsx

```diff
@@ -21,96 +21,57 @@
+
+import ReadMoreText from '../../common/ReadMoreText';
+import { Theme } from '../../theme';
 import { useThemeContext } from '../../theme/ThemeContext';
 
-//Styles
 const rowStyle = (index: number) => css`
 	background-color: ${index % 2 === 0 ? '' : '#F5F7F8'};
 `;
 
-const tdStyle = css`
+const tdStyle = (theme: Theme, cellIndex: number, rowIndex: number) => css`
+	${theme.typography.data}
 	padding: 12px;
-	border-bottom: 1px solid #eaeaea;
 	max-width: 30vw;
 	white-space: pre-wrap;
 	overflow-wrap: break-word;
 	word-break: break-word;
 	vertical-align: top;
+	${cellIndex === 0 &&
+	`
+		position: sticky;
+		left: 0;
+		z-index: 10;
+		background-color: ${rowIndex % 2 === 0 ? 'white' : '#F5F7F8'};
+	`}
+	border: 1px solid #DCDDE1;
 `;
-const linkStyle = css`
-	color: #0b75a2;
-	font-size: 12px;
-	cursor: pointer;
-	margin-left: 6px;
-
-	&:hover {
-		text-decoration: underline;
-	}
-`;
-const chevronDownStyle = css`
-	margin-left: 4px;
-`;
-
-const chevronUpStyle = css`
-	margin-left: 4px;
-	transform: rotate(180deg);
-`;
-
-// Component Implementation
 
-type TableRowProps<T> = {
+export type TableRowProps<T> = {
 	row: Row<T>;
 	index: number;
 };
 
 const TableRow = <T,>({ row, index }: TableRowProps<T>) => {
-	const [expandedCells, setExpandedCells] = useState<Record<string, boolean>>({});
 	const theme = useThemeContext();
-	const { ChevronDown } = theme.icons;
-
-	const toggleExpand = (cellId: string) => {
-		setExpandedCells((prev) => ({
-			...prev,
-			[cellId]: !prev[cellId],
-		}));
-	};
-
 	return (
 		<tr key={row.id} css={rowStyle(index)}>
-			{row.getVisibleCells().map((cell) => {
-				const cellValue = cell.getValue();
-
-				if (cellValue === undefined || cellValue === null || cellValue === '') {
-					return <td key={cell.id} css={tdStyle}></td>;
-				}
-
-				const valueStr = cellValue.toString(); // concatenate a large string to test dropdown feature
-				const isLong = valueStr.length > 68; // Length for truncation, to best match the Figma design
-				const isExpanded = expandedCells[cell.id];
-
-				if (isLong) {
-					return (
-						<td key={cell.id} css={tdStyle}>
-							<span>{isExpanded ? valueStr : valueStr.slice(0, 68) + ' ...'}</span>
-							<span css={linkStyle} onClick={() => toggleExpand(cell.id)}>
-								{!isExpanded ? '\nRead more' : '\nRead less'}
-								<ChevronDown
-									style={!isExpanded ? chevronDownStyle : chevronUpStyle}
-									fill="#000"
-									width={10}
-									height={10}
-								/>
-							</span>
-						</td>
-					);
-				}
-
+			{row.getVisibleCells().map((cell, cellIndex) => {
 				return (
-					<td key={cell.id} css={tdStyle}>
-						{flexRender(cell.column.columnDef.cell, cell.getContext())}
+					<td key={cell.id} css={tdStyle(theme, cellIndex, index)}>
+						<ReadMoreText
+							expandedText="Show Less"
+							collapsedText="Show All"
+							wrapperStyle={() => css`
+								${theme.typography.data}
+							`}
```

Comment:
curious why typography styling isn't in the ReadMoreText component?
"wrapperStyle" props are usually to allow a parent component to tweak something.